### DISCUSSION TITLE
Png flagged as malware (trojan)

### DISCUSSION THREAD

**Discussion by TypicalGhostGuy22** [Tyrrrz/DiscordChatExporter#1307] [Q&A]
One of the Pngs downloaded to assests got flagged as malware, could that actually be a trojan in a form of a image or do you think its falsely flagged??

**Comment by Tyrrrz** [comment]
Hard to say. DCE just downloads whatever assets are referenced by the chat log, some of which are served from 3rd-party sources. So theoretically it's not impossible for it to be a trojan.
